source/about/mattermost-mobile-releases.md

```diff
@@ -21,6 +21,7 @@ We strongly recommend using the latest mobile app release available that contain
 
 | **Release** | **Released on** | **Support ends** | **Compatible with** |
 |:---|:---|:---|:---|
+| v2.27 {ref}`FEATURE <release-v2-27-0>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.27.0) \| {ref}`Changelog <release-v2-27-0>` | 2025-04-16 | 2025-05-15 | {ref}`v10.7 <release-v10.7-feature-release>`, {ref}`v10.6 <release-v10.6-feature-release>`, {ref}`v10.5 <release-v10-5-extended-support-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>` |
```

Comment:
@amyblais - I'm seeing a mix of periods and dashes in link bookmarks: ``<release-v10.7-feature-release>`` vs ``<release-v10-5-extended-support-release>``. No concerns at all if it's intentional (feature vs ESR). If it's not intentional, I recommend either periods or dashes for both the releases and changelog pages.